readme.md

```diff
@@ -368,6 +368,28 @@ import bourne from '@hapijs/bourne';
 })();
 ```
 
+##### fetch
+
+Type: `Function`\
+Default: `fetch`
+
+User-defined `fetch` function.
```

Comment:
I think we should mention that it *has* to be fully compatible with https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API